airflow/providers/google/suite/hooks/drive.py

```diff
@@ -180,12 +180,25 @@ def get_file_id(self, folder_id: str, file_name: str, drive_id: Optional[str] =
             file_metadata = {"id": files['files'][0]['id'], "mime_type": files['files'][0]['mimeType']}
         return file_metadata
 
-    def upload_file(self, local_location: str, remote_location: str) -> str:
+    def upload_file(
+        self,
+        local_location: str,
+        remote_location: str,
+        chunk_size: int = 104857600,
```

Comment:
Why this magic number?

Reply:
It's the default chunk size from the original MediaFileUpload class: 100 x 1024 x 1024

Reply:
Thanks. Maybe add a comment explaining this?

Reply:
I'm trying to understand this part also: 

> :param chunk_size: File will be uploaded in chunks of this many bytes. Only
            used if resumable=True. Pass in a value of -1 if the file is to be
            uploaded as a single chunk. Note that Google App Engine has a 5MB limit
            on request size, so you should never set your chunksize larger than 5MB,
            or to -1.

If the chunk size shouldn't be larger than 5MB, why would they put 100MB as default?

Reply:
> If the chunk size shouldn't be larger than 5MB, why would they put 100MB as default?

It's only Google App Engine, not Google Drive.

Reply:
> Thanks. Maybe add a comment explaining this?

Write it as `100*1024*1024` please too .

Reply:
It's written like this in the hook's `download_file` method. I think we should keep it like this and change all of them afterwards if necessary.

Reply:
NA... Change it straight away - it make more sense to explicitly show where it is from. And parser will anyhow optimize it and calculate.

BTW. Also "chunksize" is not liked by Sphinx it seems :). The  `chunk size` would be better.

Reply:
done